docs/machine-learning/ml-data-exploration.md

```diff
@@ -48,8 +48,9 @@ Key objectives of the exploration workshops include the following:
 1. Understand and document the features, location, and availability of the data.
 2. What order of magnitude is the current data (e.g., GB, TB)? Is this all relevant?
 3. How does the organization decide when to collect additional data or purchase external data? Are there any examples of this?
-4. What data has been used so far to analyse recent data-driven projects? What has been found to be most useful? What was not useful? How was this judged?
-5. What additional internal data may provide insights useful for data-driven decision-making for proposed projects? What external data could be useful?
-6. What are the possible constraints or challenges in accessing or incorporating this data?
-7. How was the data collected? Are there any obvious biases due to how the data was collected?
-8. What changes to data collection, coding, integration, etc has occurred in the last 2 years that may impact the interpretation or availability of the collected data
+4. Understand the quality of the data. Is there already a data validation strategy in place?
+5. What data has been used so far to analyse recent data-driven projects? What has been found to be most useful? What was not useful? How was this judged?
```

Comment:
NIT: Perhaps split this up:
5. Previous data usage:
a. Which data used ... ? 
b. What has been found  ... ?

>